fe/projects/Seller.Portal/src/shared/components/Catalog/Catalog.js

```diff
@@ -225,79 +400,62 @@ function Catalog(props) {
-                                            </TableRow>
-                                        ))}
-                                    </TableBody>
+                                    <DragDropContext
+                                        onDragEnd={(result) => onDragEnd(result)}
+                                        onDragStart={(result) => onDragStart(result)}
+                                    >
+                                        <Droppable
+                                            droppableId="categories"
+                                            mode="virtual"
+                                            renderClone={(provided) => (
+                                                tableRow(provided, draggingItem)
+                                            )}
+                                        >
+                                            {(providedDroppable) => (
+                                                <TableBody
+                                                    ref={providedDroppable.innerRef}
+                                                    {...providedDroppable.droppableProps}
+                                                >
+                                                    {items.map((item, index) => (
+                                                        !isDragableDisable && draggingItem.id && placeholderProps.position == index ?
+                                                            <TableRow height={placeholderProps.height} /> :
+                                                            <Draggable
+                                                                key={item.id}
+                                                                draggableId={item.id}
+                                                                index={index}
+                                                                isDragDisabled={isDragableDisable}
+                                                            >
+                                                                {(providedDraggable) => (
+                                                                    tableRow(providedDraggable, item)
+                                                                )}
+                                                            </Draggable>
+                                                    ))}
+                                                </TableBody>
+                                            )}
+                                        </Droppable>
+                                    </DragDropContext>
                                 </Table>
                             </TableContainer>
                         </div>
+                        {!isDragableDisable && draggingItem.id &&
+                            <div className="is-flex is-flex-centered">
+                                <div
+                                    className="catalog__change-page-aera"
```

Comment:
**suggestion:** You can look on bulma docs for flexbox: https://bulma.io/documentation/columns/sizes/